src/main/view/StyleManager.cpp

```diff
@@ -7,10 +7,43 @@
 
 #include "StyleManager.h"
 
+#include <algorithm>
+
+#if defined(_WIN64) || defined(_WIN32)
+//Windows specific code
```

Comment:
I think it is generally better to make things overly clear, you never know what we might forget over the years to come.